sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -999,6 +1041,35 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setValidate(false).build();
     }
 
+    /** Choose the method used to write data to BigQuery. See the Javadoc on {@link Method} for
+     * information and restrictions of the different methods.
+     */
+    public Write<T> withMethod(Method method) {
+      return toBuilder().setMethod(method).build();
+    }
+
+    /** Choose the frequency at which file writes are triggered.
+     *
+     * <p>This is only applicable when the write method is set to {@link Method#FILE_LOADS}.
```

Comment:
no -While true for all runners, I don't think triggering is fundamentally incompatible with bounded.